packages/components/psammead-story-promo/README.md

```diff
@@ -41,10 +45,59 @@ The `LiveLabel` component is to be used inside a `Link` in index pages to show a
 ### Props
 
 <!-- prettier-ignore -->
-| Argument  | Type | Required | Default | Example |
-| --------- | ---- | -------- | ------- | ------- |
-| service | string | yes | N/A | `'news'` |
-| dir | string | no | `'ltr'`| `'rtl'` |
+| Argument | Type   | Required | Default | Example  |
+| -------- | ------ | -------- | ------- | -------- |
+| service  | string | yes      | N/A     | `'news'` |
+| dir      | string | no       | `'ltr'` | `'rtl'`  |
+
+## IndexAlsos
+
+The Index Alsos are links to stories that are related to the top story.
+
+Within the `IndexAlsos` component there is a Visually hidden level 4 which announces text passed as prop.
+
+When there are more than one Index Alsos, they should be wrapped in a list item `IndexAlsosLi` within an unordered list `IndexAlsosUl` with the role `listitem` and `list` respectively.
+
+On the other hand, when there is exactly one Index Also, it should not be contained within a list and use the `IndexAlso` component.
```

Comment:
```suggestion
On the other hand, when there is exactly one Index Also, it should use the `IndexAlso` component and it should not be contained within a list.
```

Reply:
I think this clarifies it a little bit, previously you could read the second part of this sentence as 'should not be contained within list and should not use the IndexAlso component'.